contrib/completion/zsh/_docker

```diff
@@ -138,6 +138,14 @@ __docker_containers_names() {
     __docker_get_containers all names "$@"
 }
 
+__docker_plugins() {
+    [[ $PREFIX = -* ]] && return 1
+    integer ret=1
+    plugins=(${(f)"$(_call_program commands docker $docker_options info | sed -n "/^Plugins/,/^[^ ]/s/ $1: //p")"})
```

Comment:
Thanks you so much. Really need to learn more about this, but zsh docs are awful :(.